src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/ConfigurationPathTests.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Configuration;
+using System.IO;
+using Microsoft.DotNet.RemoteExecutor;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public class ConfigurationPathTests
+    {
+        private const string ConfigName = "APP_CONFIG_FILE";
+
+        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
```

Comment:
Might be worth running these tests against .NET Framework locally (or check that PR validation did) as that's some good indication that the fix is going to match desktop behavior.

Reply:
already done, that's how I was initially validating behavior